src/components/Layout/Header/GlobalSearch/GlobalSearchButton.tsx

```diff
@@ -79,9 +94,34 @@ export const GlobalSeachButton = () => {
     }
   })
 
+  useEffect(() => {
+    if (!searchQuery?.length) return
+
+    setSendResults([])
+    ;(async () => {
+      const parsed = await parseAddressInput({ value: searchQuery })
+      if (parsed) {
+        // Set the fee AssetId as a default - users can select their preferred token later during the flow
+        setSendResults([
+          {
+            type: GlobalSearchResultType.Send,
+            id: getChainAdapterManager().get(parsed.chainId)!.getFeeAssetId(),
+          },
+        ])
+      }
+    })()
+  }, [searchQuery])
+
   const handleClick = useCallback(
     (item: GlobalSearchResult) => {
       switch (item.type) {
+        case GlobalSearchResultType.Send: {
+          // We don't want to pre-select the asset for EVM ChainIds
+          const assetId = !isEvmChainId(fromAssetId(item.id).chainId) ? item.id : undefined
```

Comment:
cosmossdk chains for ibc assets? (honestly can't remember if we even have support for holding ibc assets or if they are just transitive during trade)

Reply:
Confirmed not needed for now since IBC assets sends on the Osmosis IBC channel aren't supported yet